src/components/build/LogContainer.tsx

```diff
@@ -105,6 +113,11 @@ export class LogContainer extends React.Component<Props, State> {
       }
     }, this.props.task.task_id, logType, this.props.task.execution);
   }
+
+  private lineContainsError = (logLine: string) => {
```

Comment:
Sorry I hadn't thought of this earlier, but there's a different API that tells us what the severity of each line is (https://evergreen.mongodb.com/json/task_log/spruce_ubuntu1604_lint_37ec5816d14e05bd02535788d114da8b97ed0231_19_07_11_18_09_18/0?type=T). I don't think it's worth prioritizing fixing at the moment, but could you add a TODO comment here about changing it?

Reply:
Done